third-party/rsutils/include/rsutils/easylogging/easyloggingpp.h

```diff
@@ -69,7 +69,7 @@
     do                                                                                                                 \
     {                                                                                                                  \
         auto logger__ = el::Loggers::getLogger( rsutils::g_librealsense_elpp_id );                                     \
-        if( logger__ && logger__->enabled( el::Level::LEVEL ) )                                                        \
+        if( logger__ && logger__->typedConfigurations() &&  logger__->enabled( el::Level::LEVEL ) )                    \
```

Comment:
Can you explain what typedConfigurations() is?

Reply:
Will we investigate how we got to a state where it's null on another ticket?

Reply:
Do we need to update_configuration when exporting to ply?

Reply:
In update_configuration I don't see we save any settings related to ply, or use any of those settings, I'll remove it